### ISSUE TITLE
train code

### ISSUE THREAD

**Issue by s0966066980** [onwn/C2N#6]
Can you provide the training code?
Want to understand the design of the loss function Lstb code
